ansible/roles/php/tasks/main.yml

```diff
@@ -14,7 +18,13 @@
 - name: Install PHP Packages
   sudo: yes
   apt: pkg={{ item }} state=latest
-  with_items: php.packages
+  with_items: '{{ php.packages }}'
   when: php.packages is defined
 
+- name: Install PHP PECL Packages
+  become: yes
+  apt: pkg={{ item }} state=latest force=yes
```

Comment:
Yes, you totally right, thank you, using variables without quotes is deprecated since some version!